charts/gateway-operator/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 0.6.0
+
+### Changes
+
+- Bump the operator's default version to 1.6.0, `kong/kubernetes-configuration` CRDs to 1.4.0 and
+  bump Gateway API CRDs to 1.3.0.
```

Comment:
We will have to adjust the Gateway API version in docs for this. 👍

Reply:
Let me create an issue for tracking